.github/workflows/build.yml

```diff
@@ -12,7 +12,11 @@ jobs:
       matrix:
         run-config:
           - { xcode_version: '10.3', simulator: 'name=iPad (5th generation),OS=12.4' }
-          - { xcode_version: '10.3', simulator: 'name=iPhone X,OS=12.4' }
+          - { xcode_version: '10.3', simulator: 'name=iPhone 8,OS=12.4' }
+          #- { xcode_version: '11.7', simulator: 'name=iPad (7th generation),OS=13.7' }
```

Comment:
nit: remove comments

Reply:
done in https://github.com/openium/KIF/commit/4f286cf6f0264c1b93b5b1f860f00ca0bfb43104